### PR TITLE
Backlog

### PR DESCRIPTION
**Pull request by davegson**
@LukeSeers please review the designs

@dhaavi, please bullshit check the texts so I'm not communicating anything wrong. I used your CC task as a template, but still I'd like your 👍

Thanks!

### CODE REVIEW

File: _data/backlog.yml [hunk 1]
```diff
@@ -1,9 +1,135 @@
...
+  - title: Full UI Redesign
+    description:
+      - Completely overhaul the Portmaster user interface to simplify all aspects of it.
+      - Easily block connections from the Monitor, easily block all connection of an app, etc...
+    projects:
+      - Portmaster
+  - title: Network Trust System
+    description:
+      - Remember which networks you trust and what security level should be used within them.
+      - Breathe some life into the Autopilot system so a switch in networks automatically changes the security level.
```
Comment (dhaavi) [inline _data/backlog.yml:16]: Sentence is a bit rough, would suggest:
```suggestion
      - Breathe some life into the Autopilot system which will automatically switch to a heightened security level in untrusted networks.
```

File: _data/backlog.yml [hunk 2]
```diff
@@ -1,9 +1,135 @@
...
+      - Portmaster
+  - title: Full UI Redesign
+    description:
+      - Completely overhaul the Portmaster user interface to simplify all aspects of it.
+      - Easily block connections from the Monitor, easily block all connection of an app, etc...
+    projects:
+      - Portmaster
+  - title: Network Trust System
+    description:
+      - Remember which networks you trust and what security level should be used within them.
```
Comment (dhaavi) [inline _data/backlog.yml:15]: There might be some misconception. Currently, saving the security level used in a network is not planned. I'm also unsure about the benefit. Could be discussed, though.
```suggestion
      - Remember which networks you trust and activate additional defenses in untrusted networks, like that Café at the corner.
```

File: _data/backlog.yml [hunk 3]
```diff
@@ -1,9 +1,135 @@
...
+    projects:
+      - Portmaster
+      - SPN
+  - title: Android Support
+    description: Support both the Portmaster and the SPN on Android.
+    projects:
+      - Portmaster
+      - SPN
+  - title: SPN Transports
+    description: Create new transport types to allow you to connect to the SPN even in restricted networks.
```
Comment (dhaavi) [inline _data/backlog.yml:106]:
```suggestion
    description:
      - Create new transport types (how you connect to the SPN) to allow you to connect to the SPN even in restricted networks.
      - These transports will be able to circumvent network level detection and are the stepping stone to the Nebula Network.
```

File: _posts/2020-01-22-how-the-spn-compares-to-tor.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ When the connection arrives at the last server of the circuit it still has to tr
 
 With the SPN however, every connection is calculated individually and leaves the network as near to the destination as possible. This means:
 
-1. your connections are spread across the globe, creating vast amounts of identities
+1. your connections are spread across the globe, using vast amounts of identities (ip addresses)
```
Comment (dhaavi) [inline _posts/2020-01-22-how-the-spn-compares-to-tor.md:42]:
```suggestion
1. your connections are spread across the globe, using vast amounts of identities (IP addresses)
```

File: _posts/2020-01-22-how-the-spn-compares-to-vpns.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ After a connection arrives at the VPN server it still has to travel to the desir
 
 With the SPN however, every connection is calculated individually and leaves the network as near to the destination as possible. This means:
 
-1. your connections are spread across the globe, creating vast amounts of identities
+1. your connections are spread across the globe, using vast amounts of identities (ip addresses)
```
Comment (dhaavi) [inline _posts/2020-01-22-how-the-spn-compares-to-vpns.md:51]:
```suggestion
1. your connections are spread across the globe, using vast amounts of identities (IP addresses)
```

**Review by dhaavi — CHANGES_REQUESTED** [review]
Reviewed content only.

**Review by LukeSeers — APPROVED** [review]

**Review by dhaavi — APPROVED** [review]

**Pull request merged**